File: dia9/medir_tiempo.py
import time


def prueba_for(numero):
    lista = []
    for num in range(1, numero + 1):
        lista.append(num)
    return lista


def prueba_while(numero):
    lista = []
    contador = 1
    while contador <= numero:
        lista.append(contador)
        contador += 1
    return lista


inicio = time.time()
prueba_for(10)
final = time.time()
print(final - inicio)
print("Intermedio")
inicio = time.time()
prueba_while(10)
final = time.time()
print(final - inicio)

File: dia9/medir_tiempo.js
function pruebaFor ( numero ) {
    const lista = [];
    for ( let num = 1; num <= numero; num++ ) {
        lista.push( num );
    }
    return lista;
}

function pruebaWhile ( numero ) {
    const lista = [];
    let contador = 1;
    while ( contador <= numero ) {
        lista.push( contador );
        contador++;
    }
    return lista;
}

// Medir el tiempo para pruebaFor
console.time( 'pruebaFor' );
pruebaFor( 10 );
console.timeEnd( 'pruebaFor' );

// Imprimir separación
console.log( "Intermedio" );

// Medir el tiempo para pruebaWhile
console.time( 'pruebaWhile' );
pruebaWhile( 10 );
console.timeEnd( 'pruebaWhile' );
